packages/config-loader/src/loader.test.ts

```diff
@@ -170,4 +172,81 @@ describe('loadConfig', () => {
       },
     ]);
   });
+
+  it('watches config files', async () => {
+    const onChange = defer<AppConfig[]>();
+    const stopSignal = defer<void>();
+
+    await expect(
+      loadConfig({
+        configRoot: '/root',
+        configPaths: [],
+        watch: {
+          onChange: onChange.resolve,
+          stopSignal: stopSignal.promise,
+        },
+      }),
+    ).resolves.toEqual([
+      {
+        context: 'app-config.yaml',
+        data: {
+          app: {
+            title: 'Example App',
+            sessionKey: 'abc123',
+            escaped: '${Escaped}',
+          },
+        },
+      },
+    ]);
+
+    await fs.writeJson('/root/app-config.yaml', {
```

Comment:
ain't this going to be a problem running the windoz tests?

Reply:
Should be good since it's mock-fs and we're not doing any path comparisons